### PR TITLE
Devise confirmation

### PR DESCRIPTION
**Pull request by ghost**
https://edu-redmine.sun-asterisk.vn/issues/32508

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: app/views/admin/courses/tab/_user.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <td><%= page_index params[:page], user_counter, Settings.per %></td>
   <td><%= user.name %></td>
   <td><%= user.email %></td>
-  <td><%= user.birthday.strftime Settings.date_format %></td>
+  <td><%= user.birthday.strftime Settings.date_format if user.birthday %></td>
```
Comment (ngocvt-0484) [inline app/views/admin/courses/tab/_user.html.erb]: user.birthday&.strftime

Comment (ghost) [reply app/views/admin/courses/tab/_user.html.erb]: Vâng ạ

**Comment by ghost** [comment]
ready

**Pull request merged**